### DISCUSSION TITLE
How to set the home page

### DISCUSSION THREAD

**Discussion by zina0316** [LawnchairLauncher/lawnchair#3141] [Q&A]
How to set the home page to another page in lawnchair?
I make my home page to the third page, but every time I restart my phone, it default go to the first page, can I fix it?

**Comment by AngelofVerdant** [comment]
I didn't understand the question, can you add more detail so I can help?

**Reply by zina0316** [reply]
the default page startup is the leftmost page, i want to set it to the middle page

**Reply by AJCxZ0** [reply]
> the default page startup is the leftmost page, i want to set it to the middle page

This is what I, too, want.

Three pages is very efficient when starting from the middle, as the other two are one swipe away in either direction.
Lawnchair does a good job by staying on the same page after lock and unlock, so the effort to move to the middle page is only needed when changing account or first unlock, but it would be nice to not have to make this effort every time.

**Comment by FreezyExp** [comment]
Is there still no way to easily edit home pages?

Set default, add, move, remove?

**Reply by FreezyExp** [reply]
I would much prefer a little home icon, to indicate the default. 
A little plus to add, a little trashcan or x to remove. 
Instead of the current (v15) failed attempt at automatic sensing that reduces customization.
Needing to add placeholder widgets just to map out the space is annoying.

**Comment by SecretSauces** [comment]
This!!!

Having the default Home page be the left-most removes the ability to swipe to more than one page with only one swipe. Please implement this feature devs!!!

**Reply by doobes** [reply]
Vigorously concur!

Absolutely. I want my "home page" to be in the middle. That way, swipe left, text messages, swipe right e-mail, etc, etc

Thanks

**Comment by GambaJo** [comment]
I have the same problem. I have 3 home screen pages. The middle on is the main screen. Left is my to do list and right my calendar agenda.

**Comment by el-pandor** [comment]
I think this would be a great feature too.

**Reply by doobes** [reply]
Unclear why this has not been implemented by now. Seems like a very logical, very simple request. Devs?

**Comment by jc259** [comment]
I'd agree with the others, please add either centermost home screen as a default option or have it user selectable.
